developer/src/kmc/src/commands/build.ts

```diff
@@ -75,12 +75,17 @@ If no input file is supplied, kmc will build the current folder.`)
         filenames.push('.');
       }
 
+      if(filenames.length > 1 && commanderOptions.outFile) {
+        // -o can only be specified with a single input file
+        callbacks.reportMessage(InfrastructureMessages.Error_OutFileCanOnlyBeSpecifiedWithSingleInfile());
+      }
```

Comment:
This is a new check that I realised was needed when I was refactoring `outFile` out of `CompilerBaseOptions`